core/src/elements/crate_item.rs

```diff
@@ -215,7 +215,10 @@ fn update_thrown_crates(
 
         let colliding_with_players = collision_world
             .actor_collisions_filtered(entity, |e| {
-                e != thrown_crate.owner && players.contains(e) && invincibles.get(e).is_none()
+                e == thrown_crate.owner
```

Comment:
This will make it _only_ kill the player who threw the crate, which isn't what we want.

What we should probably do is:

- remove this `e == thrown_crate.owner` check so that it will detect collisions with all players.
- in the loop below `for player_entity in &colliding_with_players`, we do an extra check to see if `e == thrown_crate.owner`
- if it hits the crate owner, then only kill the owner if `thrown_crate.age > 0.25`
- that way the person who throws the crate is only killed if the crate has been in the air for more than a quarter second.

You may need to mess with that 0.25 value and test it in game to make sure that's a good amount of time that keeps you from killing yourself right when you throw the item.

Reply:
ok cool, ty for the feedback. I'll start making these changes this afternoon.

Reply:
Ok I tried to implement your feedback. 

In the `for player_entity in &colliding_with_players` loop, I tried to include `if-statements` that would account for both: 
* killing the owner under the appropriate circumstances 
* killing other entities under the appropriate circumstances 

0.25 seconds seems to be a decent amount of time. 

Please let me know if you'd like me to change anything else.

Reply:
> Should the player that throws the crate live and the other player die? Or should they both survive?

They should both die.